cmd/start-amazon-cloudwatch-agent/start-amazon-cloudwatch-agent.go

```diff
@@ -92,8 +93,26 @@ func main() {
 		log.Fatalf("E! Cannot translate JSON config into TOML, ERROR is %v \n", err)
 	}
 	log.Printf("I! Config has been translated into TOML %s \n", tomlConfigPath)
+	printFileContents(tomlConfigPath)
 
 	if err := startAgent(writer); err != nil {
+		log.Printf("E! Error when starting Agent, Error is %v \n", err)
 		os.Exit(1)
 	}
 }
+
+func printFileContents(path string) {
+	file, err := os.Open(path)
+	if err != nil {
+		log.Printf("E! Error when printing file(%s) contents, Error is %v \n", path, err)
+		os.Exit(1)
+	}
+	defer func() {
+		if err = file.Close(); err != nil {
+			log.Fatal(err)
```

Comment:
I don't recall what happens in this scenario. If we `defer` here, what are the repercussions of panicking?